pingen_env/readme/USAGE.rst

```diff
@@ -0,0 +1,23 @@
+Define values under global section name called 'pingen', or in case of different config per company - global section name connected with company name:  
+
+
+[pingen.companyA]
```

Comment:
Isn't there any problem with spaces in the company name or any other special character not correctly being parsed by configuration helper?

And in any case, it should be more descriptive if you put a real demo company name:

```suggestion
[pingen.My Company (San Francisco)]
```

Reply:
agree company name maybe not the best way to differentiate, but we use it in prod env for many months already so for now ill leave it as it is.
Updated readme.

Reply:
but... please add a todo to use a tech name. This can be fragile.